File: #Task-03/Python/Sub-Task 2/Sub-Task2.py
f=open("input.txt","r")
r=f.read()
f.close()
print("The String Stored In input.txt is",r)
f=open("output.txt","w")
f.write(r)
print("The Content In input.txt Has Successfully Been Copied To output.txt")
f.close()

File: #Task-03/Java/Sub-Task 2/SubTask2.java
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileWriter;
import java.io.IOException;
import java.util.Scanner;
public class SubTask2 {
    public static void main(String[] args) {
        try{
        File inp=new File("input.txt");
        Scanner myReader=new Scanner(inp);
        while (myReader.hasNextLine()) {
            String data = myReader.nextLine();
            try{
                FileWriter ou = new FileWriter("output.txt");
                ou.write(data);
                System.out.println("Output is Written");
                ou.close();
            }catch(IOException e){
                System.out.println("Error Occured");    
            }
        }}catch (FileNotFoundException e){
            System.out.println("An error occurred.");
            e.printStackTrace();
        }
        
}
}
